samples/file-encryptor/enclave/CMakeLists.txt

```diff
@@ -21,7 +23,8 @@ target_compile_definitions(enclave PUBLIC OE_API_VERSION=2)
 target_include_directories(
   enclave
   PRIVATE ${CMAKE_CURRENT_SOURCE_DIR} # Needed for #include "../shared.h"
-          ${CMAKE_CURRENT_BINARY_DIR})
+          ${CMAKE_CURRENT_BINARY_DIR} ${CMAKE_SOURCE_DIR}
+)# Needed for "_args.h" includes
```

Comment:
Why do we need `CMAKE_SOURCE_DIR` for including `_args.h`? The header should be generated in `CMAKE_CURRENT_BINARY_DIR` (the same as the _t.c file).

BTW, The format seems wrong here.

Reply:
_args.h includes "shared.h".  That is in the CMAKE_SRC_DIR

Reply:
Which args.h you're referring to?